docs/source/algorithms/LoadPSIMuonBin-v1.rst

```diff
@@ -9,23 +9,36 @@
 Description
 -----------
 
-The algorithm LoadPSIMuonBin will read in a .bin file from the PSI
+The algorithm LoadPSIMuonBin will read in a .bin file from the PSI 
 facility, from one of the various machines that use that format.
-The file name can be absolute or relative path and should have the
+The file name can be an absolute or relative path and should have the
 extension .bin. The file should only be loaded if the first two bytes
 are "1N" representing it as the format that this will load.
 
+LoadPSIMuonBin is capable of loading in accompanying temperature files.
+There are two options for achieving this. If a temperature file path 
+is passed to the property `TemperatureFilename` it will attempt to use
+this file. With no file given and temperature file searching is not 
+turned off (on by default), this is done by setting 
```

Comment:
can we change this to something like:

 If no file is provided and the temperature file searching enabled (it is by default), then it will recursively search for the 
file in the current directory of the data file and up to 3 directories
deeper than the file.

Reply:
Done